content/steps/step-2.md

```diff
@@ -7,26 +7,29 @@ order: 2
 
 ## What is Ember?
 
-Ember or more specifically EmberJS (JS stands for **J** ava **S** cript) is a framework that helps you build entire websites using JavaScript. Many modern websites are built based on JavaScript frameworks like EmberJS alone. JavaScript websites or so-called **web applications** tend to not only feel faster to their users, but also provide much more interactivity.
+Ember or more specifically EmberJS (JS stands for **J** ava **S** cript) is a tool that helps you build entire websites using JavaScript. It itself is written in JavaScript and it provides different language patterns to you to make it more easy to manage your code and build your web app. It is also called a JavaScript **framework**, as it provides the foundation on which you can build up on your own program.
```

Comment:
I think it should either be `build up your...` or `build on your...`, `up` sounds slightly better I think

```suggestion
Ember or more specifically EmberJS (JS stands for **J** ava **S** cript) is a tool that helps you build entire websites using JavaScript. It itself is written in JavaScript and it provides different language patterns to you to make it more easy to manage your code and build your web app. It is also called a JavaScript **framework**, as it provides the foundation on which you can build up your own program.
```